website/components/LandingPage/OpenSource/index.js

```diff
@@ -43,73 +43,96 @@ function cacheGet(key) {
   }
 }
 
-function cacheSet(key, data) {
-  const twoMinutes = Date.now() + 2 * 60 * 1000;
+function cacheSet(key, data, length = 120000) {
+  const expires = Date.now() + length;
   try {
-    localStorage.setItem(
-      cacheKey(key),
-      JSON.stringify({ data, expires: twoMinutes })
-    );
+    localStorage.setItem(cacheKey(key), JSON.stringify({ data, expires }));
   } catch (error) {
     // Nothing.
   }
 }
 
-function cachedFetch(key, url, properties) {
+function cached(key, fallback, length) {
   const data = cacheGet(key);
   if (data) return Promise.resolve(data);
 
-  return fetch(url)
+  return fallback().then(data => {
+    cacheSet(key, data, length);
+    return data;
+  }, console.error);
+}
+
+function yyyymmdd(date) {
```

Comment:
How about renaming it to `formatDate`? We can add `formatNumber` too (`.toLocaleString('en-US')`).